iaso/api/org_unit_validation_status.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import viewsets, permissions
+from rest_framework.response import Response
+
+
+from iaso.models import OrgUnit
+
+
+class ValidationStatusViewSet(viewsets.ViewSet):
+    """Org unit validation status API
+
+    This API is restricted to authenticated users.
+
+    GET /api/validationstatus/
+    """
+
+    permission_classes = [permissions.IsAuthenticated]
+
+    def list(self, _):
+        return Response({"results": OrgUnit.VALIDATION_STATUS_CHOICES})
```

Comment:
I would map the choices here so you don't have to do `status[0]` in the front-end.